packages/fxa-settings/README.md

```diff
@@ -1,75 +1,182 @@
+
 ### Styling components
 
 #### Tailwind
 
-The package uses [Tailwind CSS](https://tailwindcss.com/) for the bulk of its styles. If you're not familiar you're encouraged to [learn more](https://tailwindcss.com/docs/utility-first), but the idea is simple: use predefined classes on elements to layer on individual styles. **You can accomplish most of your styling needs with the classes provided.**
+At the time of writing, the `fxa-settings`, `fxa-react`, and `fxa-admin-panel` packages are setup to share a [Tailwind CSS](https://tailwindcss.com/) configuration file found in the `fxa-react` package. If you're not familiar with Tailwind, look through [their documentation](https://tailwindcss.com/docs) to get an idea of what [utility-first](https://tailwindcss.com/docs/utility-first) (Atomic CSS) is and what you can expect while using it. The general idea is simple: use single-purpose classes on elements to layer styles until the design is achieved. **You can accomplish almost all of your styling needs with classes provided by Tailwind's default configuration or through adding them in the configuration file.**
+
+Each package has its own `tailwind.css` file. This file, any S/CSS files this file imports, and the Tailwind configuration file are compiled to produce `tailwind.out.css`. Rebuilding this file should happen automatically on any change. If you're not sure what specific class name corresponds with the style you need or if you need to check that a specific style exists in our utility classes, search through `tailwind.out.css` for these styles as needed.
 
-For example, here's what a large, centered, red-text paragraph with a blue background and thick padding would look like:
+Note that not every value from a design hand off may be available due to small variations from web tool processing (such as Sketch conversion to Invision or Figma) or because the Tailwind default config doesn't have the needed style - generally, our styles increase in **units of `4px`**. If a design reflects 17px, it's generally safe to go with the closest value divisible by 4, 16px (`1rem`), but it is up to the engineer to ask for clarification from visual design if it's not clear and to also potentially offer a strong recommendation if a design strays from the `4px` increment standard. If an exception needs to be made or if the [Tailwind default configuration](https://tailwindcss.com/docs/configuration) doesn't offer a needed value (for example, [default spacing](https://tailwindcss.com/docs/customizing-spacing#default-spacing-scale) uses `-8` to output `2rem` measurements and `-10` to output `2.5rem`, but if we need `2.75rem`, it's completely acceptable to need `-9`), use the `extends` option in the configuration file to add the class that you need. We overwrite some of the default values provided by Tailwind, such as color and breakpoints. Refer to the shared Tailwind config file to see how values vary.
```

Comment:
> Refer to the shared Tailwind config file to see how values vary.

I know you mentioned where it was located a couple paragraphs ago, but it might be useful to link to the config file somewhere here or above.